tests/backup/backup_helper.go

```diff
@@ -5434,3 +5434,128 @@ func RestartAllVMsInNamespace(namespace string, waitForCompletion bool) error {
 	}
 	return nil
 }
+
+// UpgradeKubevirt upgrades the kubevirt control plane to the given version
+// If workloadUpgrade is set to true, kubevirt workloads are upgraded to the given version
+func UpgradeKubevirt(versionToUpgrade string, workloadUpgrade bool) error {
+	k8sKubevirt := kubevirt.Instance()
+	k8sCore := core.Instance()
+
+	// Checking current version
+	current, err := k8sKubevirt.GetVersion()
+	if err != nil {
+		return err
+	}
+	log.Infof("Current version is - %s", current)
+
+	// Compare and validate the upgrade path
+	currentKubevirtVersionSemVer, err := version.NewSemver(strings.TrimSpace(strings.ReplaceAll(current, "v", "")))
+	if err != nil {
+		return err
+	}
+	versionToUpgradeSemVer, err := version.NewSemver(strings.TrimSpace(strings.ReplaceAll(versionToUpgrade, "v", "")))
+	if err != nil {
+		return err
+	}
+	if currentKubevirtVersionSemVer.GreaterThanOrEqual(versionToUpgradeSemVer) {
```

Comment:
Kubevirt upgrade supports upgrade from 1 version lower, we cannot upgrade from any version to any version.
You should check that current version is only 1 . version lower than upgrade version.